### PR TITLE
fix: confirmation of key update

### PR DESCRIPTION
**Pull request by ch-rfishwick**
Confirmation message for updated keys with appropriate styling

### CODE REVIEW

File: server/routes/applications.js [hunk 1]
```diff
@@ -87,6 +87,33 @@ router.get('/manage-applications/:appId/view/:env', (req, res, next) => {
   });
 });
 
+router.get('/manage-applications/:appId/view/:env/:keyName', (req, res, next) => {
+  logger.info(`GET request to view a single application: ${req.path}`);
```
Comment (nabeelamirch) [inline server/routes/applications.js:91]: Do any of the unite tests need updating?

File: server/routes/applications.js [hunk 2]
```diff
@@ -87,6 +87,33 @@ router.get('/manage-applications/:appId/view/:env', (req, res, next) => {
   });
 });
 
+router.get('/manage-applications/:appId/view/:env/:keyName', (req, res, next) => {
```
Comment (CH-JonathanBlake) [inline server/routes/applications.js:90]: I thought Env was being added to the end of the URL?

File: server/routes/applications.js (hunk 1 above)
Comment (ch-rfishwick) [reply server/routes/applications.js:91]: Added a unit test

File: server/routes/applications.js (hunk 2 above)
Comment (ch-rfishwick) [reply server/routes/applications.js:90]: That version is still there, this is for the specific case of having just been redirected from the update key route and needing to display the key name in the confirmation message

**Review by nabeelamirch — APPROVED** [review]

**Review by ICraig-CH — APPROVED** [review]

File: server/routes/applications.js (hunk 2 above)
Comment (ch-rfishwick) [reply server/routes/applications.js:90]: Is that too much of an impediment as far as you're concerned or are you ok if I go ahead and merge?

Comment (CH-JonathanBlake) [reply server/routes/applications.js:90]: I think we should be consistent. And its not much to change.

But I guess the sky won't fall if you merge?
Why aren't we doing this via a confirm thing the same way we did delete?

Comment (ch-rfishwick) [reply server/routes/applications.js:90]: It's a different thing. It's not confirm that you want to update, it's re-display the app overview page with a message confirming that the key was updated

Comment (ch-rfishwick) [reply server/routes/applications.js:90]: Oh hold on, are you saying you'll be ok if I do exactly the same thing but just change the orders of the request params?

Comment (ch-rfishwick) [reply server/routes/applications.js:90]: Sorry I was being silly, I thought you were objecting to the presence of this new route

Comment (ch-rfishwick) [reply server/routes/applications.js]: changed

**Review by CH-JonathanBlake — APPROVED** [review]

**Pull request merged**